src/Instrumentation/CakePHP/src/CakePHPInstrumentation.php

```diff
@@ -23,60 +16,8 @@ class CakePHPInstrumentation
     public static function register(): void
     {
         $instrumentation = new CachedInstrumentation('io.opentelemetry.contrib.php.cakephp');
```

Comment:
can you add schema URL? The latest version that will install should be 1.25 (since this package requires php 8.0+)

Reply:
I think `1.25` is the first on `php: ^8.1`, so we should use `sem-conv: 1.24` here.

Reply:
I now changed it to `TraceAttributes::SCHEMA_URL`, the same way it is done in the [PSR-18 auto-instrumentation](https://github.com/open-telemetry/opentelemetry-php-contrib/blob/c701e167156ea78ed54361318224bd00d8340325/src/Instrumentation/Psr18/src/Psr18Instrumentation.php#L30) library. I would think that this way, when installing the CakePHP auto-instrumentation, composer would install the latest compatible version of open-telemetry/sem-conv which would have the constant set to the appropriate schema url. Or am I missing something?

Reply:
That's how it should work in the future, but there is actually a moratorium on changing versions for telemetry: https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/telemetry-stability.md#schema-file-driven-telemetry-producers
We're _trying_ to follow that, but it looks like psr-18 didn't get the memo :(

So, in the meantime, we go with the latest version that we can, and we're stuck on that version for a while

Reply:
The current code in the [master](https://github.com/open-telemetry/opentelemetry-php-contrib/blob/main/src/Instrumentation/CakePHP/src/CakePHPInstrumentation.php#L25-L29) already has the value of the URL schema defined. Could you resolve current conflicts?

Reply:
> That's how it should work in the future, but there is actually a moratorium on changing versions for telemetry: https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/telemetry-stability.md#schema-file-driven-telemetry-producers We're _trying_ to follow that, but it looks like psr-18 didn't get the memo :(
> 
> So, in the meantime, we go with the latest version that we can, and we're stuck on that version for a while

I see, thanks again for the information